libs/core/algorithms/tests/unit/algorithms/adjacentdifference.cpp

```diff
@@ -24,48 +26,6 @@ void adjacent_difference_test()
     test_adjacent_difference_async(par(task));
 }
 
-template <typename IteratorTag>
-void test_adjacent_difference_exception()
-{
-    using namespace hpx::execution;
-
-    // If the execution policy object is of type vector_execution_policy,
-    // std::terminate shall be called. therefore we do not test exceptions
-    // with a vector execution policy
-    test_adjacent_difference_exception(seq, IteratorTag());
-    test_adjacent_difference_exception(par, IteratorTag());
-
-    test_adjacent_difference_exception_async(seq(task), IteratorTag());
-    test_adjacent_difference_exception_async(par(task), IteratorTag());
-}
-
-void adjacent_difference_exception_test()
-{
-    test_adjacent_difference_exception<std::random_access_iterator_tag>();
-    test_adjacent_difference_exception<std::forward_iterator_tag>();
-}
-
-template <typename IteratorTag>
-void test_adjacent_difference_bad_alloc()
-{
-    using namespace hpx::execution;
-
-    // If the execution policy object is of type vector_execution_policy,
-    // std::terminate shall be called. therefore we do not test exceptions
-    // with a vector execution policy
-    test_adjacent_difference_bad_alloc(seq, IteratorTag());
-    test_adjacent_difference_bad_alloc(par, IteratorTag());
-
-    test_adjacent_difference_bad_alloc_async(seq(task), IteratorTag());
-    test_adjacent_difference_bad_alloc_async(par(task), IteratorTag());
-}
-
-void adjacent_difference_bad_alloc_test()
-{
-    test_adjacent_difference_bad_alloc<std::random_access_iterator_tag>();
-    test_adjacent_difference_bad_alloc<std::forward_iterator_tag>();
-}
-
```

Comment:
Have these moved somewhere else or are they no longer relevant?

Reply:
Yes, they have moved to the files below: 
libs/core/algorithms/tests/unit/algorithms/adjacentdifference_exception.cpp
libs/core/algorithms/tests/unit/algorithms/adjacentdifference_bad_alloc.cpp